release.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+# defeine masked variable $GITHUB_TOKEN in gitlab webinterface: Reposetorry -> Settings -> CI/CD ->Variables
+# GITHUB_TOKEN can be crataed at https://github.com/settings/tokens (from machine user account)
+# set privileges to full repo
+# USAGE create tage (changing the name format will break things)
+# run ./releas.sh
+
+repo=Nitrokey/pynitrokey
+
+release_name=$(git describe)
+release_name="${release_name%-*-*}"
```

Comment:
Please document here, what is the expected text here.
Be advised `git describe` used here will behave differently for release tags, and will provide name without `-`. Use full invocation (did you meant `--long` here?)